src/radical/pilot/agent/agent_0.py

```diff
@@ -679,7 +701,11 @@ def _prepare_env(self, env_name, env_spec):
         pre   = env_spec.get('pre_exec') or []
         out   = None
 
-        pre_exec = '-P ". env/bs0_pre_0.sh" '
+        if env_name == 'rp':
```

Comment:
I do not understand this. `pre_exec` in both checks are identical. Is this a typo or mistake?

Reply:
Thanks - that is a leftover from an unrelated change.  Fixed.